infra/ci_fuzz.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module used by CI tools in order to interact with fuzzers.
+This module helps CI tools do the following
+  1. build fuzzers
+  2. run fuzzers
+Eventually it will be used to help CI tools determine which fuzzers to run.
+"""
+
+import argparse
+import os
+import tempfile
+
+import build_specified_commit
+
+
+def main():
```

Comment:
I don't like how we are using "name" here. Can we use the origin URL instead?

Reply:
I am working on a fix but as of right now I don't know if there is an easy way to get the repo's url from the action.

Reply:
Done.

Reply:
Jonathan and I discussed and decided it would be better to keep the repo name over the origin URL. This allows us more freedom in terms of other version control managers on top of github.